docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -300,6 +300,11 @@ The new event types are supported by the Email Event Listener.
 
 The following event types are now deprecated and will be removed in a future version: `UPDATE_PASSWORD`, `UPDATE_PASSWORD_ERROR`, `UPDATE_TOTP`, `UPDATE_TOTP_ERROR`, `REMOVE_TOTP`, `REMOVE_TOTP_ERROR`
 
+= BouncyCastle FIPS updated
+
+Our FIPS 140-2 integration is now tested and supported with version 2 of BouncyCastle FIPS libraries. This version is certified with Java 21. If you use FIPS 140-2 integration, it is recommended to
```

Comment:
Just a little comment, with the upgrade to 2.0 the bcfips is also certified in fips 140-3 (see https://www.bouncycastle.org/resources/bouncy-castle-crypto-api-attains-fips-140-3-certification/). Maybe we can change this paragraph.

Reply:
@rmartinc This is good point. The thing is, that we support also RHEL8, which itself is only FIPS 140-2, but RHEL9 is FIPS 140-3. So I've instead added additional paragraph about FIPS 140-3 when on RHEL9 system. What do you think?

Reply:
Yep, I didn't think about the JVM which should also be fips 140-3. It's OK to me.